backend/tools/import/README.md

```diff
@@ -0,0 +1,39 @@
+# PIMS Import Tool
+
+The PIMS Import Tool provides a way to parse a JSON file containing an array of items and send bundled HTTP requests to a configured API endpoint for the purpose of importing data.
+
+This console application can send any type of array of JSON objects to any API endpoint.
+
+Use this console application to import inventory data into PIMS.
+
+## Setup
+
+Create a `.env` file and populate it with the following;
+
+```
+ASPNETCORE_ENVIRONMENT=Development
+Api__Url={url to endpoint (i.e. http://pims-dev.pathfinder.gov.bc.ca/api/tools/import/properties)}
+Api__Token={JWT token}
+Import__File=./Data/properties.json
```

Comment:
Maybe add a comment that the **Data\** folder is ignored and your data won't be added to source control?